model/src/main/kotlin/config/Excludes.kt

```diff
@@ -55,6 +55,16 @@ data class Excludes(
             scopes.filter { it.matches(scope.name) } +
                     (findProjectExclude(project)?.scopes?.filter { it.matches(scope.name) } ?: emptyList())
 
+    /**
+     * Checks if [id] references a [Package] or a [Project] and calls the respective [isPackageExcluded] or
+     * [isProjectExcluded].
+     */
+    fun isExcluded(id: Identifier, analyzerResult: AnalyzerResult) =
+            analyzerResult.projects.find { it.id == id }?.let {
+                // An excluded project could still be included as a dependency of another no-excluded project.
+                isProjectExcluded(it) && isPackageExcluded(id, analyzerResult)
```

Comment:
If `it` is an excluded project then `it.id` refers to a project, so the equal `id` cannot at the same time refer to a package, so we should be able to omit the `&& isPackageExcluded(id, analyzerResult)`, or?

Reply:
The naming of the methods is a bit misleading here, `isPackageExcluded` does not check the package list of the analyzer result, but if the `id` appears in the dependency tree of a non-excluded scope. So even if a project is excluded, it could still be included as a dependency of another project.

Reply:
If have improved the docs of the function, I hope that makes it clearer.

Reply:
I had to think about it some more, but I believe I got it know: My thinking that an `id` can always only refer to either a project *or* a package is wrong. An `id` can be used to declare a project, but if that project is a dependency of another project in the same multi-project, then `id` is also used by a package reference in the tree of the dependent project.